vimeo-networking/src/main/java/com/vimeo/networking/Configuration.java

```diff
@@ -97,28 +98,49 @@ public Cache getCache() {
     }
 
     /**
-     * @deprecated use {@link #saveAccount(VimeoAccount, String)} instead
-     * <p/>
-     * We find no use in storing the password when you can persist the {@link VimeoAccount} across
-     * application sessions.
+     * Persist the {@link VimeoAccount} in the {@link AccountStore}
+     * <p>
+     * Account name is <b>required</b> for authenticated user accounts; for
+     * client credential account saving use {@link #saveNonUserAccount(VimeoAccount)}
+     *
+     * @param account     the account to persist
+     * @param accountName the name of the account.
      */
-    @Deprecated
-    public void saveAccount(VimeoAccount account, String email, String password) {
-        saveAccount(account, email);
+    public void saveAuthenticatedUserAccount(@NotNull VimeoAccount account, @NotNull String accountName) {
+        if (accountStore != null) {
+            accountStore.saveAuthenticatedUserAccount(account, accountName);
```

Comment:
_that_ is the big issue @howardrigberg and I have with the NonNull/NotNull annotations - it only generates warnings. It's on the board for us to discuss at the next Android sync.